repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl

```diff
@@ -0,0 +1,171 @@
+                    <li><a href="#resources">Resources</a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <section class="document-content jumpto-offset-left">
+
+        <section id="purpose" class="push-double--bottom">
+            <h2><@fmt.message key="purpose"/></h2>
+            <p>${indicator.purpose}</p>
+        </section>
+
+        <section id="definition" class="push-double--bottom">
+            <h2><@fmt.message key="definition"/></h2>
+            <p>${indicator.definition}</p>
+        </section>
+
+        <section id="methodology" class="push-double--bottom">
+            <h2><@fmt.message key="methodology"/></h2>
+
+            <h3><@fmt.message key="methodology"/></h3>
+            <p>${indicator.descriptor}</p>
+
+            <h3><@fmt.message key="caveats"/></h3>
+            <p>${indicator.caveats}</p>
+
+        </section>
+
+        <section id="interpretations" class="push-double--bottom">
+            <h3><@fmt.message key="interpretationGuidelines"/></h3>
+            <li>${indicator.interpretationGuidelines}</li>
+        </section>
+
+        <section id="resources" class="push-double--bottom">
+            <h3>Resources</h3>
+            <ul>
+                <li><a href="#">Sample file</a></li>
+                <li><a href="#">Sample file</a></li>
+                <li><a href="#">Sample file</a></li>
+                <li><a href="#">Sample file</a></li>
+            </ul>
+        </section>
+
+        <section id="compare" class="push-double--bottom">
+            <h2>Compare</h2>
+            <table>
+                <tr>
+                    <td>Title</td>
+                    <td>Publisher</td>
+                    <td>Published Date</td>
+                    <td>Assured Until</td>
+                </tr>
+                <tr>
+                    <td><a href="#">Cancers diagnosed via emergency routes</a></td>
+                    <td>PHE</td>
+                    <td>10 June 2017</td>
+                    <td>10 June 2020</td>
+                </tr>
+                <tr>
+                    <td><a href="#">PHE</a></td>
+                    <td>PHE</td>
+                    <td>10 June 2017</td>
+                    <td>10 June 2020</td>
+                </tr>
+                <tr>
+                    <td><a href="#">Under 75 mortality rate from cancer</a></td>
+                    <td>PHE</td>
+                    <td>10 June 2017</td>
+                    <td>10 June 2020</td>
+                </tr>
+            </table>
+        </section>
+
+        <section id="resources" class="push-double--bottom">
+            <h2>5 comments</h2>
+            <div class="panel panel--grey">
+                <p>TBC</p>
+            </div>
+        </section>
+
+    </section>
+
+
+
+
+    <section class="document-content">
+    </section>
+
+
+  <footer class="footer">
+    <div class="footer__inner">
+      FOOTER
```

Comment:
Probably just want to use the footer from base page to be consistent with the rest of the site but you can sort that later down the line